client/src/components/FAQ.tsx

```diff
@@ -0,0 +1,42 @@
+import styles from "../styles/FAQ.module.css";
+import chevron from "../assets/chevron-down.svg";
+
+import React, { useState } from "react";
+
+const FAQ = () => {
```

Comment:
Maybe `FAQs` or `FaqList` or something. Could add a `FAQ` child component to reduce to duplication.

Reply:
Not sure if I agree with this as I have followed the naming of the other files in the project.

Reply:
How do you mean? Not plural?

Reply:
Yeah they are all named in the singular with a component named the same as the file name, not sure what you want to change here lol

Reply:
That's not really a convention. Those are singular components whereas this represents a list of FAQs, or the singular name would be FAQList (or something similar). A child FAQ component could be used so that you don't repeat this for each entry:
```tsx
<div className={styles.header}>
    <strong>
        ...
    </strong>
</div>
...
```
/
```tsx
<FAQ title="...">
...
</FAQ>
```

Reply:
Also, you don't actually want to use `<strong>` here, since a title carries it's own semantic meaning and isn't emphasising anything. Use `font-weight` on the header style instead.

Reply:
So it should be `h2` I guess